index.d.ts

```diff
@@ -1226,20 +1226,27 @@ declare namespace WAWebJS {
     export type ChangeParticipantsPermissions = 
         (participantIds: Array<string>) => Promise<{ status: number }>
 
-    /** Adds or removes a list of participants by ID to the group */
-    export type ChangeGroupParticipants = 
-        (participantIds: Array<string>) => Promise<{
-            status: number;
-            participants: Array<{
-                [key: string]: {
-                    code: number
-                }
-            }>
-         } & {
-             [key: string]: number;
-         }>
-
-    export interface GroupChat extends Chat {
+    /** An object that handles the result of addParticipants method */
+    export type AddParticipantsResult = {
+        [participantId: string]: {
+            code: number;
+            message: string;
+            /** @default false */
+            isInviteV4Sent: boolean,
+        };
+    };
+
+    /** AddParticipnats options */
+    export type AddParticipnatsOptions = {
+        /** @default 500 */
+        sleep?: number;
+        /** @default true */
+        autoSendInviteV4?: boolean;
+        /** @default '' */
+        comment?: string;
+    };
+
+    export class GroupChat extends Chat {
```

Comment:
this line generated errors to me 
![image](https://github.com/pedroslopez/whatsapp-web.js/assets/8761655/95b34120-353d-4839-b3eb-4537ce802aec)


changed to interface again, is solved
![image](https://github.com/pedroslopez/whatsapp-web.js/assets/8761655/32ae826d-bc84-4277-a76c-e14d75d358c6)

Reply:
Thank you fot the note, fixed